src/lab11.py

```diff
@@ -481,38 +498,92 @@ def raster_tab(self):
         canvas.clear(skia.ColorWHITE)
         active_tab.raster(canvas)
 
-    # We need to keep this because it calls get_font
+    def plus_bounds(self):
+        plus_width = self.chrome_font.measureText("+")
+        return (self.padding, self.padding,
+            plus_width + self.padding, self.tab_header_bottom - self.padding)
+
+    def tab_bounds(self, i):
+        (plus_left, plus_top, plus_right, plus_bottom) = self.plus_bounds()
+        tab_start_x = plus_right + self.padding
```

Comment:
I know why this is here but it feels to me like inlining the relevant parts of `plus_bounds` and simplifying would be better.